package.json

```diff
@@ -36,6 +36,7 @@
     "eslint": "^8.49.0",
     "eslint-import-resolver-typescript": "^3.6.1",
     "eslint-plugin-import": "^2.29.0",
+    "eslint-import-resolver-node": "^0.3.9",
```

Comment:
(question) we need it inside the internal projects too? it yes we need to add as peer dependencies inside `packages/linters/package.json` and update instructions on `packages/linters/README.md`

Reply:
@felipefialho Even with the dependency as peerdependency I had a problem committing the file https://github.com/juntossomosmais/time-out-market/blob/main/packages/linters/src/eslint.config.react.js#L1. Basically was this error https://stackoverflow.com/questions/44865507/eslint-no-duplicates-resolve-error-unable-to-load-resolver-node.

Reply:
Uhmmm... so, make sense to keep this dependency inside project?